tests/test_select_by_mask.py

```
@@ -0,0 +1,261 @@
+    it was originally False, and we do select_by_mask(), i.e. the cache
+    should be invalidated. This is because masking can convert an
+    unsorted timeseries to a sorted timeseries.
+    If the original data is sorted, then the cached value can stay.
+    """
+
+    def test_irregular_ts(self):
+        data = IrregularTimeSeries(
+            timestamps=np.array([0.0, 1.0, 0.5, 2.0]),
+            domain="auto",
+        )
+        assert data.is_sorted() == False
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `False`; use `not data.is_sorted():` for false checks

Replace with `not data.is_sorted()`

(E712)